docs/src/references/c8y-configuration-management.md

```diff
@@ -86,6 +95,8 @@ files = [
     { path = '/etc/tedge/mosquitto-conf/c8y-bridge.conf' },
     { path = '/etc/tedge/mosquitto-conf/tedge-mosquitto.conf' },
     { path = '/etc/mosquitto/mosquitto.conf', type = 'mosquitto', user = 'mosquitto', group = 'mosquitto', mode = 0o644 }
+    { path = '/etc/child1/foo.conf', type = 'foo', childid = 'child1'  },
```

Comment:
How will this approach scale when we plan to add support for hierarchial child device? Would each layer have only one child and its all automatically agrregated at levels.

Reply:
If we need to reflect the hierarchy on the thin-edge device (i.E. the whole path from the thin-edge device to the addressed child-device), I see to options.

Assuming a hierachy:
```
    thin-edge
       |
       +--- child1
       |       +-----grandchild1_1
       | 
       +--- child2
```

Further lets assume the device we want now address is 'grandchild1_1'.

**Option 1:** the `childid` could be assigned with a list. 
Example: `childid = ['child1', 'grandchild1_1']` 

**Option 2:** we could define a delimiter (e.g. '/') that is used to separate the childids inside a string.
Example: `childid = ['child1/grandchild1_1']` 
 

In case of C8Y when each child has a unique ID, it is possible to address the grandchild directly, without any hierchy: `childid = ['grandchild1_1']`

Reply:
I think we need first to uniquely identify child devices and add support for hierarchy on top of that.

The issue on how manage path in a hierarchy is beyond what is proposed with these options. These needs to be done also for the topic names and file hierarchies.

Reply:
So far the approach is to avoid any hierarchical structure in paths, MQTT topics, etc.

For C8Y that is not a problem since the child-id (used as C8Y external ID) must be anyway unique over all hierarchy levels, and can addressed directly with their child-id (C8Y external ID).

However I will consider that comment again in PR "requirements and use-cases" (#1195).